app/isomorphic/components/molecules/forms/forgot-password.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from "react";
+import PT from "prop-types";
+
+// import { Button } from "../../atoms/button";
+import { InputField } from "../../atoms/InputField";
+// import { verifyEmailOTP, verifyEmail } from "../../helper/api";
+
+import "./forms.m.css";
+
+export function ForgotPassword({ onBackdropClick }) {
+  const [email, setEmail] = useState("");
+  const [data, setOTPData] = useState({
+    otp: "",
+    password: "",
+    confirmPassword: "",
+    id: ""
+  });
+  // const [successMsg, setSuccessMsg] = useState("");
+  const [error, setError] = useState({});
+  const [showVerficationScreen, verificationScreenHandler] = useState(false);
+
+  const emailHandler = async e => {
+    e.preventDefault();
+    e.stopPropagation();
+    console.log("need to write verify email logic");
+    // const verifiedData = await verifyEmail(email);
+    // if (verifiedData) {
+    //   data.id = verifiedData["email-token"];
+    //   verificationScreenHandler(true);
+    // } else {
+    //   setError({ message: "error" });
+    // }
+  };
+
+  const setEmailData = e => {
+    setEmail(e.target.value);
+    console.log(email);
```

Comment:
console ?

Reply:
for now email value is not used but will be used when we start form validation